core/hatch.py

```diff
@@ -62,17 +69,66 @@ def build_wandb_core(
         cwd="./core",
         env=_go_env(
             with_race_detection=with_race_detection,
+            with_cgo=with_cgo,
             target_system=target_system,
             target_arch=target_arch,
         ),
     )
 
 
-def _go_linker_flags(
-    wandb_commit_sha: Optional[str],
+def build_nvidia_gpu_stats(
```

Comment:
do you think it makes sense to have this in the nvidia_gpu_stats folder? i.e. it's own hatch file?

Reply:
this should definitely be in its own `hatch.py` file as it's not building `wandb-core`